.github/workflows/php-js-e2e-tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
             - name: Set up PHP
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: 7.4
+                  php-version: '8.0'
```

Comment:
Mm I only noticed this now. Where these changes here before? 🤔 

Should we set up our test environment to have PHP 8? This could cause flaky tests as we have different dev and test envs. It literally happened to me a few days ago that I was using a feature only available in PHP 8, but as we should support lower PHP versions as well, I needed to remove it.

Reply:
Good catch. I think this one accidentally slipped into this PR. I reset both `.github/workflows/php-js-e2e-tests.yml` and `.github/workflows/unit-tests.yml`.